farm/evaluation/metrics.py

```diff
@@ -189,11 +206,41 @@ def squad_f1_single(pred, label, pred_idx=0):
     f1 = (2 * precision * recall) / (precision + recall)
     return f1
 
+
+def confidence(preds):
+    conf = 0
+    for pred in preds:
+        conf += pred[0][0].score
+    return conf/len(preds) if len(preds) else 0
+
+
+def metrics_per_bin(preds, labels):
+    pred_bins = [[] for _ in range(10)]
+    label_bins = [[] for _ in range(10)]
+    count_per_bin = [0]*10
+    for (pred, label) in zip(preds, labels):
+        current_score = pred[0][0].score
+        if current_score == 1.0:
```

Comment:
Do we need to use `math.isclose` or `numpy` function with `eps =  10**-5`

Reply:
`current_score` should never be larger than 1 but I can change the if statement to `if current_score == 1.0:` to be sure.